vnext/src/Libraries/Components/AccessibilityInfo/legacySendAccessibilityEvent.windows.js

```diff
@@ -21,7 +21,7 @@ function legacySendAccessibilityEvent(
   if (eventType === 'focus') {
     UIManager.sendAccessibilityEvent(
       reactTag,
-      UIManager.getConstants().AccessibilityEventTypes.typeViewFocused,
+      UIManager.getConstants().FocusAccessibilityEvent,
```

Comment:
I think this should be kept as AccessibilityEventTypes.typeViewFocused, and have the native module interpret that as it will. That reduces divergence between the windows JS and core JS

Reply:
hm I'm not sure how that works though, I see AccessibilityEventTypes is defined like this?
```js
type AccessibilityEventTypes = 'focus';
```


so what is typeViewFocused?

Reply:
It seems they have their native code for constants set up slightly differently where they have a getConstants method that returns a Map where AccessibilityEventTypes is a key which contains another map where typeViewFocused is a key. FB Source: [here](https://github.com/facebook/react-native/blob/fa644272360f8bd166937e83d4fcb05bb61f514a/ReactAndroid/src/main/java/com/facebook/react/uimanager/UIManagerModuleConstants.java#L111-L119)

Reply:
`constants.Add(L"AccessibilityEventTypes",...);` method for ReactConstantsProvider won't let me define a constant unless the value is just an int, double, string, etc. If I try to have the constant be a collection of event types either through an enum, or a map, or a struct with constant values, the build fails.

Reply:
thanks; yes this should get better once we start doing codegen, right now everything is a bit too fragile